### PR TITLE
WA bills: update to action_text to include text of action links

### PR DESCRIPTION
**Pull request by chrisyamas**
Scraper updated to include text of links for each action in the `action_text` stored as value of `description` attribute in bill object's `add_action()` method.

**Example using prior session bill:**
- Before this commit, [HB 1841 (from the 2021-22 session)](https://app.leg.wa.gov/billsummary?BillNumber=1841&Initiative=false&Year=2021) had it's second `action_text` stored as:
   - "First reading, referred to Finance."
- With this commit, that `action_text` will include the text of the link associated with it and be stored as:
   - "First reading, referred to Finance. (View Original Bill)"

*Note*: the scope of this commit was limited to grabbing the text. A future commit will get the `href` for the links, so that they can be also stored during the execution of bill object's `add_action()` method.

### CODE REVIEW

**Review by bfossen-ce — APPROVED** [review]
LGTM

File: scrapers/wa/bills.py [hunk 1]
```diff
@@ -447,16 +447,14 @@ def scrape_actions(self, bill, chamber, bill_url, prefile_year, second_year):
 
             rows = header.xpath("following-sibling::div[1]/div")
             for row in rows:
+                # skip later lines that are just links to files
                 if row.xpath("div[1]")[0].text_content().strip() != "":
```
Comment (johnseekins) [inline scrapers/wa/bills.py:451]: `if not`?

Comment (chrisyamas) [reply scrapers/wa/bills.py:451]: Good catch, I didn't go looking for syntax improvements of the original code during my initial commit. I'll give it another look and push a new commit with changes.

Comment (chrisyamas) [reply scrapers/wa/bills.py]: [New commit](https://github.com/openstates/openstates-scrapers/pull/4315/commits/c20c240503b1354d3845ffb5640f9bb9fdd31181) with change to this and other logic, syntax, and regex improvements (including following your prior guidance about pre-compiling all regular expressions).

**Review by NewAgeAirbender — APPROVED** [review]

**Pull request merged**